.travis.yml

```diff
@@ -15,6 +15,7 @@ rvm:
   - 2.5.1
 env:
   global:
+    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="~> 6.0"
```

Comment:
I need to figure out a way to not duplicate these environment variables. I think I just need to add `GEM_BOLT=true` as its own yaml array element. I think It is OK to merge for now with the duplicate.